provers/risc0/guest/src/mem.rs

```diff
@@ -33,3 +33,8 @@ pub unsafe extern "C" fn calloc(nobj: usize, size: usize) -> *mut c_void {
 pub unsafe extern "C" fn free(_size: *const c_void) {
     // Intentionally a no-op, since the zkvm allocator is a bump allocator
 }
+
+#[no_mangle]
+pub extern "C" fn __ctzsi2(x: u32) -> u32 {
```

Comment:
why we need this one?

Reply:
Some issue with compiling secp256k1 if I remember correctly where it's undefined similar to the malloc stuff, but I think Cecilia's stuff will change this again so didn't give it too much thought for now, I think likely the code path isn't even triggered but not sure.